examples/addons/cluster-autoscaler/README.md

```diff
@@ -23,7 +23,7 @@ The following is an example:
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
-      "orchestratorRelease": "1.10",
+      "orchestratorRelease": "1.12.8",
```

Comment:
`orchestratorRelease` is the version without the patch, so this should be `"1.12".